src/main/java/roomescape/dto/ReservationResponse.java

```diff
@@ -0,0 +1,15 @@
+package roomescape.dto;
+
+import java.time.format.DateTimeFormatter;
+import roomescape.model.Reservation;
+
+public record ReservationResponse(Long id, String name, String date, String time) {
+
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+
+    public static ReservationResponse from(final Reservation reservation) {
+        final String date = reservation.getDate().format(DateTimeFormatter.ISO_DATE);
+        final String time = reservation.getTime().format(TIME_FORMATTER);
+        return new ReservationResponse(reservation.getId(), reservation.getName(), date, time);
+    }
```

Comment:
약간 억지(?)일수도 있다고 생각하는데, ISO_DATE 보다는 직접 글자로 명시해주는게 더 직관적일수 있다 생각하는데, 레디는 어떻게 생각해?

```suggestion
    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");

    public static ReservationResponse from(final Reservation reservation) {
        final String date = reservation.getDate().format(DATE_FORMATTER);
        final String time = reservation.getTime().format(TIME_FORMATTER);
        return new ReservationResponse(reservation.getId(), reservation.getName(), date, time);
    }
```